CONTRIBUTING_DOCS.md

```diff
@@ -132,27 +132,50 @@ Examples:
+
+- If you are adding a new page to the current release directory, add the following metadata just under the `title` of the page: `redirect_from: latest/<path-to-new-file>`. This ensures that readers who use `latest` in their paths instead of the version number of the current release get redirected to your new page. If you do not add this, you may experience `htmlproofer` errors.
+
+
+### Deleting and renaming pages
+
+If you need to delete or rename a directory or file:
+
+- Ensure that you [adjust the side navigation bar to match](#side-navigation-bar).
+
+- If the page exists in previous versions, add the following metadata just under the `title` of the files in the directories of the previous versions: `no-canonical: true`. Otherwise, `htmlproofer` will fail.
+
+### Side navigation bar
+
+The naming and layout of the side navigation bar is stored in `_data/$VERSION/navbars/*`. Jekyll automatically stores information from the `_data` directory in an accessible variable called `site.data`. The top-level layout (`_layout/docwithnav.html`) will iterate through all the files in `site.data[version].navbars` to construct the sidebar based on which version is being viewed.
 
 > **Note**: Sidebar paths to index files (see [next section](#linking-content)) should end in a `/` in the yaml file. Sidebar paths to actual files should not end in a `/` in the yaml file.
 
-URL structure is important. In order to create a top level splash page for a URL path, simply name the file `index.md`. See the following example:
 
+## Link syntax
+
+### Closing slashes
 
-| URL                                           | Filepath                                         |
-|-----------------------------------------------|--------------------------------------------------|
-| `/getting-started/kubernetes/`                | `/getting-started/kubernetes/index.md`           |
-| `/getting-started/kubernetes/troubleshooting` | `/getting-started/kubernetes/troubleshooting.md` |
+To link to a page not named `index.md`, omit the closing slash. To link to a page named `index.md`, include a closing slash. See the following table for some examples.
 
 
-## Linking content
+| URL                                           | File path                                         |
+|-----------------------------------------------|---------------------------------------------------|
+| `/getting-started/kubernetes/`                | `/getting-started/kubernetes/index.md`            |
+| `/getting-started/kubernetes/troubleshooting` | `/getting-started/kubernetes/troubleshooting.md`  |
 
-All links should be absolute links. To link to versioned content, prefix all links with: `{{site.baseurl}}/{{page.version}}/`
+
+### Relative links and `page.version` variable
+
+To link to other doc site content, use relative links prefixed with: `/{{page.version}}/`. This allows the content to port across documentation versions without link breakage.
```

Comment:
an example might be nice: `See [this page]({{site.baseurl}}/{{page.version}}/reference/public-cloud/gce)`

Reply:
>relative links prefixed with: `/`

You say relative but `page.version` is usually only used with absolute links...

Reply:
Works great with relative links also.